schema/snapcraft.yaml

```diff
@@ -224,10 +224,10 @@ properties:
   apps:
     type: object
     additionalProperties: false
-    validation-failure:
-            "{!r} is not a valid app name. App names consist of upper- and
+    validation-failure: |
```

Comment:
by using | you are saying, use this string verbatim.

Reply:
perhaps a better choice would be `>`.  ???

Reply:
yes, but let's go with the json PR